test/widget_tests/after_auth_screens/app_settings/app_setting_page_test.dart

```diff
@@ -90,14 +85,14 @@ Future<void> main() async {
   locator<GraphqlConfig>().test();
   locator<SizeConfig>().test();
 
-  final Directory dir = Directory('temporaryPath');
-  Hive
-    ..init(dir.path)
-    ..registerAdapter(UserAdapter())
-    ..registerAdapter(OrgInfoAdapter());
-  await Hive.openBox<User>('currentUser');
-  await Hive.openBox<OrgInfo>('currentOrg');
-  await Hive.openBox('url');
+  // final Directory dir = Directory('temporaryPath');
+  // Hive
+  //   ..init(dir.path)
+  //   ..registerAdapter(UserAdapter())
+  //   ..registerAdapter(OrgInfoAdapter());
+  // await Hive.openBox<User>('currentUser');
+  // await Hive.openBox<OrgInfo>('currentOrg');
+  // await Hive.openBox('url');
```

Comment:
Why we need to make these changes??. I don't think we need these changes in this PR.

Reply:
It was failing the test otherwise at my local machine

Reply:
But  it is not failing in any other PR??